.github/CheckStatus/CheckStatus.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
Need copyright header.

Reply:
Which should it be? I didn't find copyright headers in this repo, which I could use as a reference.